install/kubernetes/helm/istio/values.yaml

```diff
@@ -32,37 +22,16 @@ galley:
 #
 # mixer configuration
 #
+# @see charts/mixer/values.yaml
 mixer:
-  policy:
-    enabled: true
-  telemetry:
-    enabled: true
-  # mixer load shedding configuration.
-  # When mixer detects that it is overloaded, it starts rejecting grpc requests.
-  loadshedding:
-    # disabled, logonly or enforce
-    mode: enforce
-
-  adapters:
-    # stdio is a debug adapter in istio-telemetry, it is not recommended for production use.
-    stdio:
-      enabled: false
-  resources:
-    requests:
-      cpu: 1000m
-      memory: 1G
-    limits:
-      # It is best to do horizontal scaling of mixer using moderate cpu allocation.
-      # We have experimentally found that these values work well.
-      cpu: 4800m
-      memory: 4G
+  enabled: true
 
 #
 # pilot configuration
 #
+# @see charts/pilot/values.yaml
 pilot:
   enabled: true
-  traceSampling: 1.0
 
```

Comment:
why are we removing trace sampling by default?

Reply:
It is moved down into the subchart instead of specifying it 2 times.

Reply:
ok SG